src/app/components/content-page/content-page.component.html

```diff
@@ -1,12 +1,20 @@
-<header class="fixed-top d-flex">
+<header class="fixed-top d-flex flex-column">
   <nav class="d-flex flex-grow-1 p-2">
     <div>
       <app-profile-indicator></app-profile-indicator>
     </div>
 
-    <div id="search" class="ml-auto">s</div>
+    <button type="button" class="img-button my-auto ml-auto" (click)="isCollapsed = !isCollapsed"
```

Comment:
At the onclick event should we not put this rather in a function? Keeping logic out of the html page and in the typescript file which should contain any logic of a view.

Reply:
this is bootstrap, used to toggle the search bar

Reply:
Aaa cool just checking :)